.travis.yml

```diff
@@ -1,45 +1,86 @@
 sudo: required
-language: python
-
-branches:
-  only:
-  - master
-  - devel
+language: minimal
 
 git:
-  depth: 3
+  depth: 2
 
 services:
   - docker
 
 env:
-  - DOCKER_COMPOSE_VERSION=1.12.0
+  global:
+    - IMAGE_REPO=gcr.io/dd-decaf-cfbf6/model
```

Comment:
Just curious: Why does the repo name contains cfbf6?

Reply:
It's just a randomly generated part of the google cloud project id which we cannot control.